sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/aio/_form_recognizer_client_async.py

```diff
@@ -0,0 +1,180 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+# pylint: disable=protected-access
+
+from typing import (  # pylint: disable=unused-import
+    Any,
+    List,
+    IO,
+    TYPE_CHECKING,
+)
+import six
+from azure.core.tracing.decorator_async import distributed_trace_async
+from azure.core.polling.async_base_polling import AsyncLROBasePolling  # pylint: disable=no-name-in-module,import-error
+from .._generated.aio._form_recognizer_client_async import FormRecognizerClient as FormRecognizer
+from .._policies import CognitiveServicesCredentialPolicy
+from .._response_handlers import (
+    prepare_receipt_result,
+    prepare_layout_result
+)
+from .._generated.models import AnalyzeOperationResult
+from .._helpers import get_content_type, POLLING_INTERVAL
+if TYPE_CHECKING:
+    from .._credential import FormRecognizerApiKeyCredential
+    from .._models import (
+        ExtractedReceipt,
+        ExtractedLayoutPage
+    )
+
+
+class FormRecognizerClient(object):
+    """FormRecognizerClient.
+
+    :param str endpoint: Supported Cognitive Services endpoints (protocol and hostname,
+        for example: https://westus2.api.cognitive.microsoft.com).
+    :param credential: Credentials needed for the client to connect to Azure.
+        This is an instance of FormRecognizerApiKeyCredential if using an API key.
+    :type credential: ~azure.ai.formrecognizer.FormRecognizerApiKeyCredential
+    """
+
+    def __init__(
+            self,
+            endpoint: str,
+            credential: "FormRecognizerApiKeyCredential",
+            **kwargs: Any
+    ) -> None:
+        self._client = FormRecognizer(
+            endpoint=endpoint,
+            credential=credential,
+            authentication_policy=CognitiveServicesCredentialPolicy(credential),  # TODO: replace with core policy
+            **kwargs
+        )
+
+    @distributed_trace_async
+    async def begin_extract_receipts(self, stream: IO[bytes], **kwargs: Any) -> List["ExtractedReceipt"]:
```

Comment:
Should this be singular, `begin_extract_receipt`?

Reply:
We can extract multiple, different receipts. Seems like `receipts` is appropriate, is there a reason not to pluralize? I think .NET has it plural as well.

Reply:
It is plural in .NET as well.  For every extract operation, multiple forms can be returned.